ksql-serde/src/main/java/io/confluent/ksql/serde/json/KsqlJsonDeserializer.java

```diff
@@ -57,17 +76,12 @@
       .put(Type.BYTES, KsqlJsonDeserializer::enforceValidBytes)
       .build();
 
-  private final Gson gson;
```

Comment:
Do we no longer need Gson?  We may be able to drop it as a dependency if that is the case...